README.md

```diff
@@ -1,34 +1,18 @@
 # linter-htmlhint
 
-This linter plugin for [Linter](https://github.com/AtomLinter/Linter) provides
-an interface to [htmlhint](https://github.com/yaniswang/HTMLHint). It will be
+A plugin for [Atom Linter](https://github.com/AtomLinter/Linter) providing an interface to [HTMLHint](https://github.com/yaniswang/HTMLHint). It will be
 used with files that have the syntax.
 
 ## Installation
 
-Linter package must be installed in order to use this plugin. If Linter is not
-installed, please follow the instructions [here](https://github.com/AtomLinter/Linter).
-
-### Plugin installation
-
-```ShellSession
+```bash
 $ apm install linter-htmlhint
 ```
 
-### .htmlhintrc
+## Config
 
-This plugin will search for a htmlhint configuration file called `.htmlhintrc`
-and use that file if it exists anywhere in the directory tree. It will stop at
-the first `.htmlhintrc` file found.
+This plugin will search for a [HTMLHint](https://github.com/yaniswang/HTMLHint) configuration file called `.htmlhintrc` and use that file if it exists anywhere in the directory tree. It will stop at the first `.htmlhintrc` file found.
```

Comment:
In any parent folder, not "anywhere in the directory tree" (as that implies anywhere in the filesystem at all).